src/widget/wmainmenubar.cpp

```diff
@@ -51,7 +52,21 @@ QString showPreferencesKeyBinding() {
 #endif
 }
 
+QUrl documentationFileUrl(const QString& resourcePath, const QString& fileName) {
+    QDir resourceDir(resourcePath);
+#if !defined(__APPLE__)
```

Comment:
Can you add a todo/fixme why we're excluding macos?

Reply:
I don't know the reason why it was like that, but I can re-add the previous comment
```
    // FIXME: We don't include the PDF manual in the bundle on OSX.
    // Default to the web-hosted version.
```